tox.ini

```diff
@@ -10,8 +10,8 @@ deps = .[test,ceph,gnocchi]
        http://tarballs.openstack.org/swift/swift-master.tar.gz#egg=swift
        python-swiftclient
        tooz[redis]
-       http://tarballs.openstack.org/keystone/keystone-master.tar.gz
        httpbin
+       py36,py37,py38: http://tarballs.openstack.org/keystone/keystone-master.tar.gz
```

Comment:
you can also use `!py27` which might be handier.

Reply:
Indeed :)